autoload/neomake.vim

```diff
@@ -528,17 +530,16 @@ function! neomake#GetEnabledMakers(...) abort
     return enabled_makers
 endfunction
 
-function! s:HandleLoclistQflistDisplay(file_mode) abort
-    let open_val = get(g:, 'neomake_open_list')
-    if open_val
+function! s:HandleLoclistQflistDisplay(open_val, file_mode) abort
```

Comment:
I do want to to have the settings maker-dependant: i.e., some makers autoopen the Quickfix, others not. I can transfer those properties to jobinfo afterwards ?